css/style.css

```diff
@@ -74,12 +79,151 @@ body {
     padding: 0;
   }
 
-  .connect-me {
+  .connect-two {
     padding-top: 12px;
-    padding-bottom: 15px;
     font-weight: 500;
     font-size: 16px;
     line-height: 24px;
     text-decoration: none;
   }
+
+  #works {
+    display: grid;
+  }
+
+  .work-card {
+    background-color: #fff;
+    margin: 114px 24px 0 24px;
+    padding: 16px;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+  }
+
+  .experience {
+    display: flex;
+    align-items: center;
+    gap: 20px;
+    margin-top: 12px !important;
+  }
+
+  .categories {
+    display: flex;
+    align-items: center;
+    gap: 8px;
+    padding: 0;
+  }
+
+  .categories li {
+    list-style-type: none;
+    background: #ebebff;
+    border-radius: 8px;
+    text-align: center;
+    padding: 7px 10px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  .work-details > h2 {
+    margin: 12px 0;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+  }
+
+  .experience .title {
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: #344563;
+  }
+
+  .experience p {
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #7a869a;
+  }
+
+  .work-desc {
+    margin: 15px 0;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .btn-primary {
```

Comment:
- [ ] Please follow the style guidelines for the buttons (Active and Hover). 😄 
![image](https://user-images.githubusercontent.com/92215878/191948833-aeb58a69-27a0-4b11-8eb6-53a8e3fe103b.png)